file-store/src/lib.rs

```diff
@@ -94,6 +94,14 @@ impl FileStore {
         Ok(())
     }
 
+    pub fn update_total_size(&self) -> Result<u64> {
```

Comment:
the naming of that function is a bit confusing to me.

I parsed it as "please update the total size please", initially, while it means "get the total size of the update files".

I know prefixing with `get` is unidiomatic in Rust, but in the interest of avoiding ambiguity, I wonder if we wouldn't want `get_update_total_size` here. Or even, given this is not a free operation and always recomputed from scratch, maybe use a verb to convey this information? `compute_update_total_size` or `process_update_total_size`?

Reply:
Yeah I like the compute, I’m going to update both function